ee/apps/omnichannel-transcript/package.json

```diff
@@ -19,6 +19,7 @@
 		"@rocket.chat/emitter": "0.31.22",
 		"@rocket.chat/model-typings": "workspace:^",
 		"@rocket.chat/models": "workspace:^",
+		"@rocket.chat/pdf-worker2": "workspace:^",
```

Comment:
```suggestion
		"@rocket.chat/pdf-worker": "workspace:^",
```

Reply:
This would also require renaming other files :see_no_evil: but I don't think the suffix makes sense here

Reply:
Ohhhh, this is a "legacy" thing lol. The pdf-worker was the microservice, when i made it a package, i put it as "pdf-worker2" and seems i didn't change lol